spec/apis/v1/submission_comments_api_spec.rb

```diff
@@ -188,6 +205,102 @@ def annotation_notification_call(author_id: @student.to_param, assignment_id: au
       @course = @course.root_account.courses.create!(workflow_state: 'available')
       annotation_notification_call(author_id: @student.to_param)
       expect(response.status).to eq 404
+
```

Comment:
It looks like two `end` statements got removed here, so when I try to run the test suite it's throwing an error. looks like we just need an `end` to close out this spec, and an `end` to close out the `describe 'annotation_notification' do` on line 83.

However, I did ocally add those `end` statements back in so I could run the tests, and I'm seeing the following spec failure:

```
Failures:

  1) Submissions Comment API #destroy user has permission to delete the comment deletes the submission comment
     Failure/Error: expect(@submission.submission_comments).to be_empty
       expected `#<ActiveRecord::Associations::CollectionProxy [#<SubmissionComment id: 427, comment: "Hello world!", ... nil, draft: false, edited_at: nil, attempt: nil, root_account_id: 1967, workflow_state: "active">]>.empty?` to return true, got false
     # ./spec/apis/v1/submission_comments_api_spec.rb:304:in `block (4 levels) in <top (required)>'
     # ./spec/support/spec_time_limit.rb:46:in `block in enforce'
     # ./spec/support/spec_time_limit.rb:45:in `enforce'
     # ./spec/spec_helper.rb:368:in `block (2 levels) in <top (required)>'

Finished in 22.86 seconds (files took 42.84 seconds to load)
25 examples, 1 failure

Failed examples:

rspec ./spec/apis/v1/submission_comments_api_spec.rb:289 # Submissions Comment API #destroy user has permission to delete the comment deletes the submission comment

Randomized with seed 58809
```